Tests/AnimationReadyTests.swift

```diff
@@ -0,0 +1,76 @@
+import XCTest
+
+@testable import Lottie
+
+@MainActor
+final class AnimationReadyTests: XCTestCase {
+
+  // MARK: Internal
+
+  func test_dotLottieFile_hasAnimation_onInitialize() {
+    XCTExpectFailure("This is expected failure due to loading delay")
```

Comment:
[Note] This is the underlying issue, that is different from `test_jsonFile_hasAnimation_onInitialize`